corehq/apps/export/static/export/js/list_exports.ng.js

```diff
@@ -80,6 +80,11 @@
             var currentUrl = useLegacyBulkExportUrl ? $scope.legacy_bulk_download_url : $scope.bulk_download_url;
             input.closest("form").attr("action", currentUrl);
         };
+        $scope.downloadRequested = function($event) {
```

Comment:
Is the `$` prefix an angular convention? It made me think `$event` was a jQuery object for a second.